### ISSUE TITLE
`MOUSEINPUT.mouseData` type is `i32` instead of win32's `DWORD`/`u32`

### ISSUE THREAD

**Issue by lifers** [microsoft/windows-rs#2631]
### Summary

Description says it all, using `windows` crate version `0.51.1`. Is there any reason why it is implemented as such? Thanks in advance.

Reference: https://learn.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-mouseinput

**Comment by kennykerr** [comment]
Yep, that doesn't line up with what's in the Windows headers. The `windows` crate gets this type information from metadata. I'll transfer this issue to the metadata repo for resolution.
